### PR TITLE
[minor] link to a tutorial for the #[implement(...)] macro

### PR DESCRIPTION
**Pull request by daladim**
Tiny doc addition as a follow-up to https://github.com/microsoft/windows-rs/issues/2882

### CODE REVIEW

File: crates/libs/implement/src/lib.rs [hunk 1]
```diff
@@ -7,6 +7,9 @@ use quote::{quote, ToTokens};
 /// Implements one or more COM interfaces.
 ///
 /// # Example
+///
+/// Note that you could also have a look at [this more complete tutorial](https://kennykerr.ca/rust-getting-started/how-to-implement-com-interface.html)
```
Comment (kennykerr) [inline crates/libs/implement/src/lib.rs:11]:
```suggestion
/// Here is a [more complete tutorial](https://kennykerr.ca/rust-getting-started/how-to-implement-com-interface.html).
```

**Review by kennykerr — APPROVED** [review]

**Comment by riverar** [comment]
I'm not a fan of increasing the number of pointers to Kenny's personal blog/domain, but not going to block this.

**Comment by kennykerr** [comment]
We'll hopefully move the tutorials I wrote over to windows-rs soon.

**Review by riverar — APPROVED** [review]
Looks good as-is. We have some future work queued up to improve the docs situation 👍

**Pull request merged**